pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -346,6 +346,14 @@ async def run(self):
                 if not self.skip_mirror_binaries:
                     message_digests = await self.extract_and_publish_clients(client_type, release_infos)
                 if not self.skip_signing:
+                    if not message_digests:
```

Comment:
I think this means we never sign on the first promote that gets this far, and always sign on the second promote that gets this far.

Reply:
if mirror step not skipped the message digest list exist then will signing them.